src/codegen.zig

```diff
@@ -372,11 +372,32 @@ pub fn generateSymbol(
             return Result{ .appended = {} };
         },
         .Struct => {
+            // TODO debug info
+            // TODO padding of struct members
```

Comment:
I'll help with this in a future patch :)

Reply:
Thanks! It shouldn't be too hard as `Type.structFieldOffset()` already provides a lot of relevant information I think.